vhdl_lang/src/analysis/declarative.rs

```diff
@@ -606,7 +617,14 @@ impl<'a, 't> AnalyzeContext<'a, 't> {
                     scope.add(ent, diagnostics);
                 }
             }
+            Declaration::PackageDeclaration(ref mut unit) => {
+                self.analyze_package(unit, diagnostics)?;
+            }
+            Declaration::PackageBody(ref mut unit) => {
+                self.analyze_package_body(unit, diagnostics)?;
```

Comment:
This (and the package declaration analysis) is fine, but I think that this declares them in the global namespace. As I have written in the second comment, check that this is correct.

Reply:
I will try to resolve this next tomorrow or so. It is not exactly easy to interpret what the LRM really means, I have to say lol.

Reply:
Yea, one of the hardest parts of developing a VHDL language server is interpreting the LRM :P